aten/src/ATen/native/cuda/stochastic_rounding.cuh

```diff
@@ -0,0 +1,67 @@
+// Ref: https://gitlab.com/riship11/stochastic-rounding
```

Comment:
Make sure to remove all references to Apex and Nvidia internal repos.

Reply:
This is a public gitlab repository (we can access from anywhere) thus I thought it's allowed to refer.

Reply:
anyway remove this reference.